test/support/pages/DiscoverPage.js

```diff
@@ -0,0 +1,199 @@
+// in test/support/pages/DiscoverPage.js
+define(function (require) {
+  // the page object is created as a constructor
+  // so we can provide the remote Command object
+  // at runtime
+  var Common = require('./Common');
+
+  var defaultTimeout = 20000;
+  var common;
+  var thisTime;
+
+  function DiscoverPage(remote) {
+    this.remote = remote;
+    common = new Common(this.remote);
+    thisTime = this.remote.setFindTimeout(defaultTimeout);
+  }
+
+  DiscoverPage.prototype = {
+    constructor: DiscoverPage,
+
+    clickTimepicker: function clickTimepicker() {
+      return thisTime.findByClassName('navbar-timepicker-time-desc').click();
+    },
+
+    clickAbsoluteButton: function clickAbsoluteButton() {
+      return thisTime.findByLinkText('Absolute').click();
+    },
+
+    setFromTime: function setFromTime(timeString) {
+      return thisTime.findByCssSelector('input[ng-model=\'absolute.from\']')
+      .type('\b' + '\b' + '\b' + '\b' + '\b' + '\b' + '\b' +
```

Comment:
I tried clearValue and couldn't get it to work.  But I'll try again and see if I missed something my first try.